Examples/Tests/photon_photon_collisions/analysis_photon_photon_collisions.py

```diff
@@ -0,0 +1,200 @@
+        V = (xmax-xmin)*(ymax-ymin)*(zmax-zmin)
+        return E1, E2, theta, dt, V
+
+def is_close(val1, val2, rtol=default_tol, atol=0.):
+    # wrapper around numpy.isclose, used to override the default tolerances
+    return np.isclose(val1, val2, rtol=rtol, atol=atol)
+
+# check that the photons have been completely transformed into pairs:
+# because the fusion multiplier is 1, as soon as a linear Breit-Wheeler event occurs,
+# the two photons must disappear and 2 electron-positron pairs must be generated
+def check_final_macroparticles(test):
+    if test == 'two_photons_test':
+        w1, w2 = get_input_parameters(test)
+        macro_photonA_number = np.loadtxt('diags/reducedfiles/ParticleNumber.txt')[-1,3]
+        macro_photonA_weight = np.loadtxt('diags/reducedfiles/ParticleNumber.txt')[-1,8]
+        macro_photonB_number = np.loadtxt('diags/reducedfiles/ParticleNumber.txt')[-1,4]
+        macro_photonB_weight = np.loadtxt('diags/reducedfiles/ParticleNumber.txt')[-1,9]
+        assert(macro_photonA_number == macro_photonB_number == 0.)
+        assert(macro_photonA_weight == macro_photonB_weight == 0.)
+        macro_positron_number = np.loadtxt('diags/reducedfiles/ParticleNumber.txt')[-1,6]
+        macro_positron_weight = np.loadtxt('diags/reducedfiles/ParticleNumber.txt')[-1,11]
+        macro_electron_number = np.loadtxt('diags/reducedfiles/ParticleNumber.txt')[-1,5]
+        macro_electron_weight = np.loadtxt('diags/reducedfiles/ParticleNumber.txt')[-1,10]
+        assert(macro_electron_number == macro_positron_number == 2.)
+        assert(macro_electron_weight == macro_positron_weight == w1 == w2)
+
+def check_energy_conservation():
+    ekin_data = np.loadtxt('diags/reducedfiles/ParticleEnergy.txt')
+    ekin_photonA = ekin_data[:,3]
+    ekin_photonB = ekin_data[:,4]
+    ekin_electron = ekin_data[:,5]
+    ekin_positron = ekin_data[:,6]
+    num_data = np.loadtxt('diags/reducedfiles/ParticleNumber.txt')
+    num_phys_electron = num_data[:,10]
+    num_phys_positron = num_data[:,11]
+    field_energy = np.loadtxt('diags/reducedfiles/FieldEnergy.txt')[:,2]
+    total_energy = ekin_photonA + ekin_photonB + ekin_electron + ekin_positron + m_e*c**2*(num_phys_electron + num_phys_positron) + field_energy
+    assert(np.all(is_close(total_energy, total_energy[0], rtol=5e-12)))
+
+def check_momentum_conservation():
+    total_momentum_x = np.loadtxt('diags/reducedfiles/ParticleMomentum.txt')[:,2]
+    total_momentum_y = np.loadtxt('diags/reducedfiles/ParticleMomentum.txt')[:,3]
+    total_momentum_z = np.loadtxt('diags/reducedfiles/ParticleMomentum.txt')[:,4]
+    assert(np.all(is_close(total_momentum_x, total_momentum_x[0])))
+    assert(np.all(is_close(total_momentum_y, total_momentum_y[0])))
+    assert(np.all(is_close(total_momentum_z, total_momentum_z[0])))
+
+def check_charge_conservation():
+    rho_max = np.loadtxt('diags/reducedfiles/RhoMaximum.txt')[:,2]
+    assert(is_close(rho_max, rho_max[0]).all())
```

Comment:
Related to the comment above, could this be simply replaced by the following?
```suggestion
    assert(np.all(np.isclose(rho_max, rho_max[0], rtol=default_tol, atol=0.)))
```